packages/ember-htmlbars/tests/helpers/each_test.js

```diff
@@ -607,6 +612,127 @@ test("it supports {{itemViewClass=}} with in format", function() {
 
 });
 
+test("it supports {{emptyView=}}", function() {
```

Comment:
Still failing... I could use some help on this one. The problem seems to be that the below `view` can't find its container when the parent view is creating it the empty view instance ([see here](https://github.com/emberjs/ember.js/blob/d1d5578cbefda479851db99074ab959671351794/packages/ember-views/lib/views/view.js#L1924-L1930)).

If I tweak the below view (see below), it fails deeper when its attempting to append to the document:

```javascript
view = EmberView.create({
  template: templateFor('{{each view.people emptyView="anEmptyView"}}'),
  people: A(),
  container: container
});
```

Ping @mmun